pkgs/servers/nextcloud/packages/28.json

```diff
@@ -119,16 +119,6 @@
       "agpl"
     ]
   },
-  "integration_paperless": {
```

Comment:
Why was this removed?

Reply:
Just my own stupidity, sorry!

(regenerated on 24.05, but the app isn't packaged there).